### ISSUE TITLE
Can we adjust/zoom pop-up window like the one in firefox ?

### ISSUE THREAD

**Issue by vijaykarani333** [polywock/globalSpeed#122]
While this plugin is available in Firefox, but seem like the PiP feature is not visible on youtube.com. 
So in order to get the most out of this plugin, I currently use Edge, but facing a limitation after paying video in PiP mode . (not sure if this is a chromium (since chrome can't  do this as well) OR youtube limitation). 

In Firefox, after paying in the PiP mode. We can still adjust the widow size, which sometimes provides better visibility of the content. For example. Although we can't really see what the exponent  is after the PiP mode, but we can still adjust the size of the Pop-out window,

![image](https://user-images.githubusercontent.com/69556286/132521931-2a45d0b5-1da8-40ac-92ea-df501bcb6eae.png)

or zoom within the same window to enlarge the content. like 

<img width="845" alt="image" src="https://user-images.githubusercontent.com/69556286/132522818-1a9d5841-2ee9-49b8-86d1-24b35d885bb2.png">
or adjusting the size of the pop-out window.

<img width="1190" alt="image" src="https://user-images.githubusercontent.com/69556286/132523125-77692521-00c9-4fe5-8de6-309345a8c31e.png">

Can use achieve any one of these two features in the chrome or edge while using this plugin ? (not choosing Firefox for another reason also understand that Firefox and Chrome/Edge is build with different "technologies" ).  

Thank you.

**Comment by polywock** [comment]
Hey Vijay. Global Speed doesn't really have a PiP feature. Clicking on PiP button would just trigger the browser's native PiP mode through the `HTMLVideoElement.requestPictureInPicture()` API. Firefox doesn't support that API yet. Nowadays, it's not as useful because Chrome's new Global Media Controls does the same thing. 

![b](https://user-images.githubusercontent.com/31208859/132533021-9e2b6d45-847e-4bdb-b5d8-4507aa25dc19.png)

Even so, I keep it around for a few reasons. 
1. Unlike Chrome, Edge don't have Global Media Controls enabled by default yet and require users to enable a flag, so might still be useful for them.  
2. Some websites like DIsney, Hulu, etc block PiP mode by adding a `disablePictureInPicture` attribute to videos. If the website is blocking PiP mode, the Global Media Controls won't allow the user to enter PiP mode. But, Global Speed's PiP button will remove the attribute and force PiP mode. 
3. On some websites Global Speed can detect several videos and hotkeys like rewind/forward/pause may trigger the incorrect video. For these cases, Global Speed lets the user select which video they want prioritized for hotkeys by clicking the arrow icon. The PiP button can help them tell which video is the correct one. 

![c](https://user-images.githubusercontent.com/31208859/132536694-f8b9bceb-7e84-4dc5-ba18-74f2e05cea4a.png)

In short, Global Speed's PiP button isn't really special and I can't control how PiP videos look or zoom.
